wiki/Glossary/en.md

```diff
@@ -619,15 +610,7 @@ If the player runs out of osu!coins, they may purchase more—out of their own w
 - 999USD = 1000 osu!coins (0.001% savings!)
 - first unborn child = infinite osu!coins (unborn child will work for osu!)
 
-The following day, peppy had "Rollback osu!coins due to popular demand" and ends with this [remark](https://disq.us/p/w2ydos).
-
-### osu!cookie
-
-![](img/Intro_static.jpg)
-Yes, that big circle that reads "osu!" is the osu!cookie
-
-The osu!cookie, also known as "cookie", is another word for "osu!logo" which has appeared many times in the osu!client.
-The osu!cookie is seen in the main menu, in solo mode, and once in old crash reports (seen [here](img/Pippi_corruption.jpg)).
+The following day, peppy had "Rollback osu!coins due to popular demand" and ends with [this remark](https://disq.us/p/w2ydos).
```

Comment:
"The following day, peppy rolled back the implementation of osu!coins due to popular demand, and ended with.."